python/ray/serve/_private/constants.py

```diff
@@ -264,6 +264,14 @@
 RAY_SERVE_CONTROLLER_CALLBACK_IMPORT_PATH = get_env_str(
     "RAY_SERVE_CONTROLLER_CALLBACK_IMPORT_PATH", None
 )
+# The method name on the serve deployment which will return the custom autoscaling metrics to the controller.
+RAY_SERVE_AUTOSCALING_STATS_METHOD = get_env_str(
```

Comment:
we can probably just make this hardcoded instead of allowing it to be configured by env var